package.json

```diff
@@ -4,11 +4,9 @@
   "homepage": "https://liamjohnston.github.io/comic-slack/",
   "private": true,
   "dependencies": {
-    "node-sass": "^4.12.0",
     "react": "^16.3.2",
     "react-copy-to-clipboard": "^5.0.1",
     "react-dom": "^16.3.2",
-    "react-scripts": "3.2.0",
```

Comment:
Any idea why this is removed? This is part of create-react-app, right...? And `node-sass`?

Reply:
Semantically they should be in `dev` but looks like the convention is that these packages need to be in `dependencies` I've just reverted that.